packages/styles/src/switch/switch.scss

```diff
@@ -0,0 +1,80 @@
+@import "../core/_dev";
+
+.as-switch {
```

Comment:
I miss some disabled styles and the disabled example in Catalog. I don't think the current disabled styles match specification.

Reply:
fixed